src/shortest_path.jl

```diff
@@ -0,0 +1,107 @@
+"""
+	shortest_path!(
+		model,
+		g, source, target, edge_cost, var_name
+	)
+
+Modify a JuMP model by adding the variable, constraints and objective necessary to compute the shortest path between 2 vertices over a graph
+
+The edge selection variable will be named `var_name`
+"""
+function shortest_path!(
+    model::Model,
+    g::AbstractGraph,
+    source::Int,
+    target::Int,
+    edge_cost::AbstractMatrix;
+    var_name,
+)
+    dg = g
+    if !is_directed(g)
+        dg = DiGraph(g)
```

Comment:
not necessary, but test with undirected graphs

Reply:
Removed and tested, it should be fine without it